spec/tapioca/cli/check_shims_spec.rb

```diff
@@ -5,7 +5,7 @@
 
 module Tapioca
   class CleanShimsTest < SpecWithProject
-    describe "tapioca clean-shims" do
+    describe "# cli::clean-shims" do
```

Comment:
I tried but it's not, maybe because of the reporter we use, but the output is always folded:

Example:
```
describe "a" do
  describe "b" do
    describe "c" do
       # ..
    end
  end
end
``` 

Gives the output: `a::b::c`.

Since it's not used in the output let's not add useless blocks in our tests.